warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexCachingIteratorJexl.java

```diff
@@ -888,20 +902,38 @@ protected boolean isTimedOut() {
     /**
      * Get a source copy. If the source is setup as a ThreadLocalPooledSource, then no copy is needed.
```

Comment:
fix comment

Reply:
perhaps add comment that this should only be used for the unsorted case

Reply:
done.